.travis.yml

```diff
@@ -1,23 +1,29 @@
 # http://travis-ci.org/#!/jupyter/jupyterhub
 language: python
-sudo: false
 python:
-    - 3.7-dev
-    - 3.6
-    - 3.5
-    - 3.4
     - 3.3
+    - 3.4
+    - 3.5
+    - 3.6
+    - nightly
```

Comment:
I think it's better to use the descending order, because nightly will often take the longest, so it should start first.